File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/WindowAnchor.java
package com.spvessel;

import com.spvessel.Core.InterfaceWindowAnchor;

public class WindowAnchor extends Prototype implements InterfaceWindowAnchor {
    private static int count = 0;

    public WindowAnchor() {
        setItemName("WindowAnchor_" + count);
        count++;
    }
}

File: CSharp_docOnly/Items/WindowAnchor.cs
using System.Drawing;
using SpaceVIL.Core;

namespace SpaceVIL
{
    public class WindowAnchor : Prototype, IWindowAnchor
    {
        static int count = 0;
        public WindowAnchor()
        {
            SetItemName("WindowAnchor_" + count);
            count++;
            IsFocusable = false;
        }
    }
}
